api/py/ai/chronon/utils.py

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Seems like these changes are mostly linting, so not reviewing them too close. Let me know if there's actual logic changes here to TAL at.

Reply:
`join_part_name()` is where the logic was updated. The rest is linting.